readme.md

```diff
@@ -192,13 +192,18 @@ This also accepts an object with separate `connect`, `socket`, and `request` fie
 
 ###### retries
 
-Type: `number` `Function`<br>
-Default: `2`
+Type: `number` `Function` `Object`<br>
```

Comment:
I think we should remove support for `retries: Function` and just require `retries: { retry: Function} }`.

Reply:
That's not needed IMO.

Reply:
I don't think to define a custom method is that common and it would be nice to simplify the options. Right now it's quite confusing, with IMHO too many variations. Then it would just be either a number (common case) or object (full power).

Reply:
@brandon93s Thoughts?